### ISSUE TITLE
ICS File Not Showing Some Information

### ISSUE THREAD

**Issue by brianrivet** [stefanfriedrich/CraftDate2Calendar#2]
I'm having trouble getting the ICS file to include the event description and URL. Here's my code:

`{% set event = {'start'		  : entry.eventStartTime.getTimestamp(),
				'end'		  : entry.eventEndTime.getTimestamp(),
				'title'		  : 'Private Event - ' ~ entry.contactFirstName ~ ' ' ~ entry.contactLastName,
				'description' : entry.eventDescription | striptags,
				'location'	  : 'Walk-Ons - ' ~ entry.locationName,
				'url'         : entry.url } %}`

The ICS file link works and I can download the file and add it to my calendars, but the description and URL are not included in the event information when it loads into my calendar. Can you tell me what I might be doing wrong, or if this is not possible?

**Comment by stefanfriedrich** [comment]
hi, could you attach the ics-file? thanks

**Comment by brianrivet** [comment]
Hi Stefan,

I was able to figure out the issues. I was referencing an incorrect variable for the description, so that is why the description wasn’t showing up. However, on the URL, when I looked at the plugin file, it didn’t look like anything was in place for the url. I added some code to make it work. I’ve attached the files here. If it was supposed to output the url before, I didn’t see where it was set up to do that.

Thanks!

Brian Rivet
brian@odysseydesignstudio.com
Phone: 225-505-9243
Fax: 225-300-8517
www.odysseydesignstudio.com
